### DISCUSSION TITLE
Daily Yield is not matching with Huawey Solar

### DISCUSSION THREAD

**Discussion by kaciker** [wlcrs/huawei_solar#275] [General]
Hello
I write this here, because I'm not usre if this is a bug, or is a concept  missunderstanding from my side
I have in the App Huawei Solar the value. "Yield" in the example 19,75Kwh  and I have from this integration the value "Daily Yield" in the example 13,18kwh, at the same momment. I never realized until now, tht both values are not the same.

Need to be the same ? if not, what is the value in the integration for this value in Huawei Solar ?
I checked by modbus the register 32114, and match with the Huawei  SOlar website, for that example, 19,75kwh

Can you help me to understand it ? 

![image](https://user-images.githubusercontent.com/7668181/214647797-e094659c-3b3b-40e7-b484-51ebe51d5ad8.png)

**Comment by tdr-its42** [comment]
You'll find some background information in the wiki:
https://github.com/wlcrs/huawei_solar/wiki/Daily-Solar-Yield

**Reply by kaciker** [reply]
thanks, is clear ¡
